### ISSUE TITLE
The SVGs have hard-coded paths

### ISSUE THREAD

**Issue by moonglum** [innoq/tabelle#9]
The SVGs for the arrows have hard-coded paths, so if you install tabelle as an NPM package, the images won't be found:

https://github.com/innoq/tabelle/blob/master/components/atoms/radio-arrow/_radio-arrow.scss#L7

This needs to be configurable somehow 🤔

**Comment by joyheron** [comment]
Good point. I'm wondering if it would be a good idea here to use SVG sprites, but I'm not sure exactly how that works. I will look into it.
